tests_selector/tests/evaluation/eval_helpers.py

```diff
@@ -0,0 +1,120 @@
+"""Helper functions for evaluation code"""
+import subprocess
+import random
+from tests_selector.utils.git import (
+    get_git_repo,
+)
+
+
+def checkout_remove_branch():
```

Comment:
Looks very similar to `TestHelper.checkout_new_branch()`, shouldn't it be used here with branch name as a parameter?